src/validation.h

```diff
@@ -58,8 +58,6 @@ namespace Consensus {
 struct Params;
 } // namespace Consensus
 
-/** Default for -limitancestorcount, max number of in-mempool ancestors */
-static const unsigned int DEFAULT_ANCESTOR_LIMIT = 25;
 /** Default for -limitancestorsize, maximum kilobytes of tx + all in-mempool ancestors */
```

Comment:
The rest should be moved too, as stated in the comment.

    DEFAULT_ANCESTOR_LIMIT
    DEFAULT_ANCESTOR_SIZE_LIMIT
    DEFAULT_DESCENDANT_LIMIT
    DEFAULT_DESCENDANT_SIZE_LIMIT
    DEFAULT_MEMPOOL_EXPIRY